ambry-router/src/test/java/com.github.ambry.router/MockKeyManagementService.java

```diff
@@ -34,6 +35,12 @@ public SecretKeySpec getKey(short accountId, short containerId) throws GeneralSe
     if (exceptionToThrow.get() != null) {
       throw exceptionToThrow.get();
     } else {
+      if (timeToBlockResponse.get() != null) {
+        try {
+          Thread.sleep(timeToBlockResponse.get());
+        } catch (InterruptedException e) {
```

Comment:
Any reason we're swallowing the exception here?

Reply:
its just a mock and when close is called while waiting, this is expected to throw. If it had been a prod code, we wouldn't have swallowed